chunkcache.cpp

```diff
@@ -55,9 +55,9 @@ ChunkCache& ChunkCache::Instance() {
 
 void ChunkCache::clear() {
   QThreadPool::globalInstance()->waitForDone();
-  mutex.lock();
+
+  QMutexLocker guard(&mutex);
```

Comment:
using QMutexLocker is in general the more robust apporach as it also unlocks the mutex in case of an exception or a return.

Reply:
That is obviously right.